### DISCUSSION TITLE
UV Index Data?

### DISCUSSION THREAD

**Discussion by fullychargedmarketing** [weather-gov/api#414] [Q&A] [answered]
Does the NWS API provide any UV Index data?

**Comment by JJKraw** [comment, marked as answer]
I get the data from the EPA.

https://www.epa.gov/enviro/web-services

Scroll down to "UV Index".

**Reply by fullychargedmarketing** [reply]
This is "exactly" what I needed.

**Comment by reidcooper** [comment]
I'd love to have the ability to consolidate my API calls to this service and to have it include the current UV Index. That would be fantastic.
